extra/write_throughput_benchmark.rst

```diff
@@ -37,14 +37,13 @@ Once we create a new EC2 instance, we need to install pgbench on this instance.
 
 If you are running a **Debian** based system, simply type::
 
+  sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+  sudo apt-get install wget ca-certificates
+  wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
   sudo apt-get update
   sudo apt-get install postgresql-10
 
-If you are running a **RedHat** based system, simply type::
-
-  sudo yum update
-  sudo yum install postgresql10-contrib
-
+If you are running a **RedHat** based system, follow the `yum installation guide <https://wiki.postgresql.org/wiki/YUM_Installation>`_.
```

Comment:
For Debian systems I was able to write a single script that worked. However the systems using the `yum` package manager vary too much for a simple script to work. Thus I point people at the postgres wiki about it.

Another option would be to drop the instructions for RHEL and just tell people to use Debian.

Reply:
Also, pgbench is available in the contrib package. We removed those specific instructions, so if someone just installs PostgreSQL, they wouldn't have pgbench?